firehose-source/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+server:
```

Comment:
Please remove this file and have your 2 defaults (doppler-url & output-json) be set via field initialization